src/services/content/index.js

```diff
@@ -7,7 +7,7 @@ const redirectHooks = require('./hooks/redirect');
 const searchHooks = require('./hooks/search');
 const materialsHooks = require('./hooks/materials');
 
-const { REQUEST_TIMEOUT } = require('../../../config/globals');
+const { Configuration } = require('@schul-cloud/commons');
```

Comment:
Codacy found an issue: [`@schul-cloud/commons` import should occur before import of `./material-model`](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045787)